app/styles/components/private/time-picker.scss

```diff
@@ -124,6 +124,7 @@
 
 						& > svg {
 							pointer-events: all;
+							width: 240px;
```

Comment:
The time picker is meant to be dynamic in width. Setting width here is not the right solution to the problem.